### ISSUE TITLE
LightGBM tests fail for 2.0.5 version

### ISSUE THREAD

**Issue by lopuhin** [TeamHG-Memex/eli5#238]
`tests/test_lightgbm.py::test_explain_weights[split]` `tests/test_lightgbm.py::test_explain_weights[weight]` fail (probably due to change of how feature importances are implemented), `tests/test_lightgbm.py::test_explain_prediction_single_leaf_tree` segfaults

**Comment by kmike** [comment]
It looks like a bug in LightGBM; I've opened an issue in their bug tracker: https://github.com/Microsoft/LightGBM/pull/847.
